content/docs/intro/cloud-providers/digitalocean/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: DigitalOcean
```

Comment:
Page titles could probably be a little more SEO-friendly - e.g. _"DigitalOcean Pulumi Provider"_. Maybe @spara has recommendations?